website/project/views/node.py

```diff
@@ -952,6 +952,11 @@ def get_summary(**kwargs):
     auth = kwargs['auth']
     node = kwargs['node'] or kwargs['project']
     rescale_ratio = kwargs.get('rescale_ratio')
+    if rescale_ratio is None and request.args.get('rescale_ratio'):
+        try:
+            rescale_ratio = float(request.args.get('rescale_ratio'))
+        except:
```

Comment:
Always explicitly catch specific `Exception` types in order to avoid errors passing silently.

```python
except (TypeError, ValueError)
```